sftp/2.1/modules/ROOT/pages/sftp-security-configuration-override.adoc

```diff
@@ -0,0 +1,64 @@
+= Configure Security Parameters of SSH Connection
+
+ The following example shows how to configure the Anypoint Connector for SFTP (SFTP Connector) to override security parameters using Anypoint Studio.
+
+== Create a Configuration Override File
+
+Follow these steps to create the file:
+
+. Navigate to your `src/main/resources` folder.
+. Create a config file, for example, `mule_sshd_config` with the following SSHD content. The file follows the format of an `sshd_config` file:
++
+[source,xml,linenums]
+----
+KexAlgorithms ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group18-sha512,diffie-hellman-group17-sha512,diffie-hellman-group16-sha512,diffie-hellman-group14-sha256
+Ciphers aes128-cbc,aes128-ctr,aes192-cbc,aes192-ctr,aes256-cbc,aes256-ctr
+HostKeyAlgorithms ssh-rsa,rsa-sha2-256,rsa-sha2-512
+MACs hmac-sha1,hmac-sha2-256,hmac-sha2-512
+----
+image::sftp-override-config-file-contents.png[Contents of the override config file]
+
+[start=3]
+. Configure the keys and values accordingly. If you do not provide any key, then default values are taken into consideration.
```

Comment:
```suggestion
. Configure the keys and values accordingly. If you do not provide a key, then default values are used.
```

Reply:
I would try to change "accordingly" to "according to [whatever it is dependent on]" if you know what that means. For the second part, I am not sure what you mean by taken into consideration. Does it evaluate them? Might say "evaluated and used."